Assets/ComethSDK/Scripts/Services/EoaFallbackService.cs

```diff
@@ -86,49 +84,58 @@ public static async Task<string> GetSignerLocalStorage(string walletAddress, str
 				walletAddress);
 
 			if (localStorageV2 == null) return null;
```

Comment:
Should not throw en error here, instead of returning null ?

Reply:
The way i see it is that its a getter, so if there isnt anything he just returns null, its not an actual error. In Get signer if the result was Null i then throw and exception "New Domain detected. You need to add that domain as signer."